src/style.css

```diff
@@ -1,3 +1,82 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+html {
+  --font-color: #00acdc;
+}
+
 body {
   margin: 0;
+  background-color: #e7e5e5;
+}
+
+.header {
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.img-link {
+  display: block;
+}
+
+.header-img {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  box-sizing: border-box;
+}
+
+.nav-list {
+  display: flex;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-item {
+  margin-left: 20px;
+}
+
+.nav-list :first-child {
+  margin-left: 0;
+}
+
+.nav-link {
+  display: block;
+  height: 60px;
```

Comment:
better to not hardcore the height, just set the paddings

Reply:
done